API/Repositories/Implementations/MatchesRepository.cs

```diff
@@ -352,7 +341,8 @@ private IQueryable<Match> MatchBaseQuery(bool filterInvalidMatches)
                 .Matches.Include(x => x.Games)
                 .ThenInclude(x => x.MatchScores)
                 .Include(x => x.Games)
-                .ThenInclude(x => x.Beatmap);
+                .ThenInclude(x => x.Beatmap)
+                .OrderBy(x => x.StartTime);
```

Comment:
The `GetAsync` method overrides this by ordering by primary key. Does this still work for the processor?

Reply:
The processor sorts by start time anyway I think, seems fine.

Reply:
Going to remove this orderby though, we shouldn't do that here anyway.